packages/core/strapi/bin/strapi.js

```diff
@@ -262,8 +262,16 @@ program
 program
   .command('transfer')
   .description('Transfer data from one source to another')
-  .addOption(new Option('--from <source>', `Source of your data`).default('local'))
-  .addOption(new Option('--to <destination>', `Destination of your data`).default('remote'))
+  .addOption(new Option('--from <sourceURL>', `URL of remote Strapi instance to get data from.`))
+  .addOption(new Option('--to <destinationURL>', `URL of remote Strapi instance to send data to`))
+  .hook('preAction', async (thisCommand) => {
+    const opts = thisCommand.opts();
+
+    if (!opts.from && !opts.to) {
```

Comment:
I can probably remove this since transfer also checks the same thing. But I don't like the idea of calling transfer at all if we know the inputs are invalid.

Reply:
I don't know if it really changes something in the end, but why not.